rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -969,14 +969,17 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
         Execution.withTransaction {
             //find any currently running executions for this job, and if so, throw exception
             def found = Execution.createCriteria().get {
+                projections {
+                    count('id', 'count')
```

Comment:
this will break because of line https://github.com/rundeck/rundeck/pull/6853/commits/59b604fff0371ae6146db3f6da8824d6358df281#diff-2be8e081cdfb61449581c41d1254854d13c2d44141d7e169407e7d22a145c21eR984